### ISSUE TITLE
QUESTION: Calibrate does not heat bed?

### ISSUE THREAD

**Issue by arshish1612** [danorder/All-In-One-V400-SR-Official-Klipper-#11]
Do we have to manually heat the bed before running the CALIBRATE macro?

**Comment by danorder** [comment]
yes to the temp you use for printing since we need to factor in thermal expansion. This is why it it'll wait for temps to stabilize per calibration cycle to avoid deviation on delta_calibrate / mesh as a further note.
